src/Assets/ugui-mvvm/BindingMode.cs

```diff
@@ -3,8 +3,8 @@
     public enum BindingMode
     {
         OneTime,
-        OneWayToView,
-        OneWayToViewModel,
+        OneWay,
```

Comment:
@ryantrem  in keeping the old ugui-mvvm BindingModes does this make sense to be "OneWayToTarget" and "OneWayToSource" to be a little more explicit ?

Reply:
Maybe... I was going back and forth in my proposal on being more explicit (e.g. OneWayToTarget) and just matching the XAML names.

Reply:
A significant number of consumers of this library might not be familiar with XAML development  so mapping every naming convention from XAML to ugui-mvvm might not be the best approach :)